src/app/screens/book-list/book-list.component.ts

```diff
@@ -9,12 +10,18 @@ import { BookService } from '@app/services/book.service';
 export class BookListComponent implements OnInit {
   books;
   filterValue = '';
+  shoppingBooks;
 
-  constructor(private bookService: BookService) { }
+  constructor(private bookService: BookService, private shoppingService: ShoppingCartService) { }
 
   ngOnInit(): void {
     this.bookService.getBooks()
       .subscribe(({page}) => this.books = page)
+      this.shoppingService.shoppingBooks.subscribe(books => this.shoppingBooks = books);
```

Comment:
you should work with all response data at the same time. Check [this](https://rxjs-dev.firebaseapp.com/api/index/function/forkJoin) 👀